src/modules/universal-discovery/components/finder/finder.component.js

```diff
@@ -186,10 +225,13 @@ export default class FinderComponent extends Component {
      * @memberof FinderComponent
      */
     onLoadMore(parentLocation) {
+        if (parentLocation === null) {
```

Comment:
if you will add the location to the `activeLocations` changes in this method are not needed.